Generator/src/main/java/microsoft/fluentui/generator/model/FluentUIIconDocument.kt

```diff
@@ -0,0 +1,102 @@
+package microsoft.fluentui.generator.model
+
+import microsoft.fluentui.generator.*
+import microsoft.fluentui.generator.toJavaEnumValueName
+import java.io.File
+import java.util.*
+import kotlin.collections.HashMap
+import kotlin.collections.HashSet
+
+class FluentUIIconDocument: LinkedHashMap<String, LinkedHashMap<String, Any>>() {
+
+    private val iconSizeMap = HashMap<String, Int>()
+    private val iconDefaults = HashMap<String, String>()
+    private val iconNameList = HashSet<String>()
+
+    fun generate() {
+        parse2()
+        ingestIcons2()
```

Comment:
why 2?

Reply:
This file is not needed. Have removed this whole file.
Marking it resolved